### PR TITLE
8297507: Update header after JDK-8297230

### PR DESCRIPTION
**Pull request by jayathirthrao**
Update header to add extra space

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8297507](https://bugs.openjdk.org/browse/JDK-8297507): Update header after JDK-8297230

### Reviewers
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**) ⚠️ Review applies to [21a54040](https://git.openjdk.org/jdk/pull/11324/files/21a54040f282bdd5158263b526d0c44774916eb0)
 * [Magnus Ihse Bursie](https://openjdk.org/census#ihse) (@magicus - **Reviewer**) ⚠️ Review applies to [21a54040](https://git.openjdk.org/jdk/pull/11324/files/21a54040f282bdd5158263b526d0c44774916eb0)
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - Author)
 * [Daniel Fuchs](https://openjdk.org/census#dfuchs) (@dfuch - **Reviewer**) ⚠️ Review applies to [21a54040](https://git.openjdk.org/jdk/pull/11324/files/21a54040f282bdd5158263b526d0c44774916eb0)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11324/head:pull/11324` \
`$ git checkout pull/11324`

Update a local copy of the PR: \
`$ git checkout pull/11324` \
`$ git pull https://git.openjdk.org/jdk pull/11324/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11324`

View PR using the GUI difftool: \
`$ git pr show -t 11324`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11324.diff">https://git.openjdk.org/jdk/pull/11324.diff</a>

</details>

### CODE REVIEW

File: src/java.desktop/share/classes/sun/java2d/marlin/DPQSSorterContext.java [hunk 1]
```diff
@@ -4,9 +4,7 @@
  *
  * This code is free software; you can redistribute it and/or modify it
  * under the terms of the GNU General Public License version 2 only, as
- * published by the Free Software Foundation. Oracle designates this
- * particular file as subject to the "Classpath" exception as provided
- * by Oracle in the LICENSE file that accompanied this code.
+ * published by the Free Software Foundation.
```
Comment (kevinrushforth) [inline src/java.desktop/share/classes/sun/java2d/marlin/DPQSSorterContext.java:7]: I don't understand this change. This file _needs_ the Classpath exception.

**Review by aivanov-jdk — APPROVED** [review]

File: src/java.desktop/share/classes/sun/java2d/marlin/DPQSSorterContext.java (hunk 1 above)
Comment (jayathirthrao) [reply src/java.desktop/share/classes/sun/java2d/marlin/DPQSSorterContext.java]: Thanks Kevin. I have updated the PR.

**Review by magicus — APPROVED** [review]

File: src/java.desktop/share/classes/sun/java2d/marlin/DPQSSorterContext.java (hunk 1 above)
Comment (aivanov-jdk) [reply src/java.desktop/share/classes/sun/java2d/marlin/DPQSSorterContext.java]: This is not an easy thing to see.

**Review by kevinrushforth — APPROVED** [review]
Presuming this passes Tier1, this looks good.

**Review by kevinrushforth — CHANGES_REQUESTED** [review]
I think you have another missing space.

**Comment by kevinrushforth** [comment]
I this this diff is also needed:

```
  * This code is distributed in the hope that it will be useful, but WITHOUT
  * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
  * version 2 for more details (a copy is included in the LICENSE file that
  * accompanied this code).
```

**Comment by aivanov-jdk** [comment]
It makes sense to copy the license header from another file in the same directory to make sure nothing's missing.

**Review by dfuch — APPROVED** [review]

**Review by kevinrushforth — COMMENTED** [review]
Updated fix looks good now.

**Comment by dcubed-ojdk** [comment]
Here's the diffs that work:
$ git diff
diff --git a/src/java.desktop/share/classes/sun/java2d/marlin/DPQSSorterContext.java b/src/java.desktop/share/classes/sun/java2d/marlin/DPQSSorterContext.java
index e2e4fdd34bf..ee924f37805 100644
--- a/src/java.desktop/share/classes/sun/java2d/marlin/DPQSSorterContext.java
+++ b/src/java.desktop/share/classes/sun/java2d/marlin/DPQSSorterContext.java
@@ -4,13 +4,13 @@
  *
  * This code is free software; you can redistribute it and/or modify it
  * under the terms of the GNU General Public License version 2 only, as
- * published by the Free Software Foundation. Oracle designates this
+ * published by the Free Software Foundation.  Oracle designates this
  * particular file as subject to the "Classpath" exception as provided
  * by Oracle in the LICENSE file that accompanied this code.
  *
  * This code is distributed in the hope that it will be useful, but WITHOUT
  * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
  * version 2 for more details (a copy is included in the LICENSE file that
  * accompanied this code).
  *

**Review by kevinrushforth — APPROVED** [review]

**Comment by kevinrushforth** [comment]
> Here's the diffs that work:

Yes, that's what Jay tested.

**Review by dcubed-ojdk — APPROVED** [review]
Thumbs up.

**Comment by jayathirthrao** [comment]
With latest change i checked validate-headers locally and it passes.

**Comment by dcubed-ojdk** [comment]
Yes, crossed in the ether...

**Comment by kevinrushforth** [comment]
> It makes sense to copy the license header from another file in the same directory to make sure nothing's missing.

Exactly. That's how I spotted the second problem -- by doing a diff with another file that wasn't touched by the PR in question.

**Pull request closed without merging**